### ISSUE TITLE
It' not work well with ranger

### ISSUE THREAD

**Issue by bommbo** [sunaku/tmux-navigate#4]
If i use ranger firstly,and then open file with vim. It not work.it maybe ranger's problem.

**Comment by sunaku** [comment]
I was able to reproduce the issue successfully:

```vim
:! tmux display -p '\#{pane_current_command}'
python

Press ENTER or type command to continue
```

The problem is that tmux sees ranger as `python`.

I will implement a fix for this sometime soon. :+1:

**Comment by sunaku** [comment]
@bommbo - Please try the [issue-4]( https://github.com/sunaku/tmux-navigate/tree/issue-4 ) branch which has a fix for this issue.

**Comment by bommbo** [comment]
> @bommbo - Please try the [issue-4](https://github.com/sunaku/tmux-navigate/tree/issue-4) branch which has a fix for this issue.

like that " set -g @plugin 'sunaku/tmux-navigate#issue-4'"? sorry,i don't know how to use that. vim-plug is easy to set.

**Comment by sunaku** [comment]
I don't know if tmux-plugins supports that syntax, but you can just do it manually:

```sh
cd ~/.tmux/plugins/tmux-navigate
git fetch origin
git checkout origin/issue-4
```

**Comment by sunaku** [comment]
Merged into main branch at commit 8327772ea72cbb7d25e636bb090b0ef7cf4a54f5.  Verified that it solves the problem. Closing.
